lib/engine.py

```diff
@@ -476,6 +475,25 @@ def exists(self, script):
             )
             for key in list(script.urls.keys()) if key])
 
+    def to_csv(self):
+        from retriever.lib.tools import sort_csv
+        if self.name is "sqlite":
+            self.connection.text_factory = str
```

Comment:
Sorry I didn't think about this before, but is it possible to set this in the `get_connection()` method for `sqlite` instead of using the conditional and setting it here?